### PR TITLE
(#826) Update Computers images and packages.config

### LINKED ISSUE #826: Update documentation for Chocolatey Central Management 0.11.0
### What You Are Seeing?
Documentation needs updating for Chocolatey Central Management 0.11.0.

### What is Expected?
All documentation is updated to reflect Chocolatey Central Management 0.11.0.

### How Did You Get This To Happen? (Steps to Reproduce)
The main documentation that will be effected lives in https://docs.chocolatey.org/en-us/central-management/

### PR DESCRIPTION
**Pull request by corbob**
## Description Of Changes

This updates the images used on the computers page to reflect the UI in Chocolatey Central Management 0.11.0. In addition, this adds new documentation about the export to packages.config feature.

## Motivation and Context

- Updated interface in Chocolatey Central Management 0.11.0.
- Addition of ability to export to a `packages.config` file.

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

Fixes #826

### CODE REVIEW

File: input/en-us/central-management/usage/website/computers.md [hunk 1]
```diff
@@ -58,8 +68,9 @@ You will be prompted to confirm the deletion.
 
 ![Prompt to confirm deletion of a computer in Central Management](/assets/images/computers/ccm-computers-delete-confirm.png)
 
+## FAQ
 
-### Computers are not showing up in CCM
+### What do I do if computers are not showing up in CCM
```
Comment (st3phhays) [inline input/en-us/central-management/usage/website/computers.md:73]:
```suggestion
### What do I do if computers are not showing up in CCM?
```

All of the FAQs on the Deployments page end with a `?`, so I've added one here.

File: input/en-us/central-management/usage/website/computers.md [hunk 2]
```diff
@@ -43,6 +43,16 @@ You will be presented with a list of the installed software packages for the mac
 
 ![Computer details screen showing installed software](/assets/images/computers/ccm-computers-details.png)
 
+### Exporting a `packages.config` file for a computer
+
+> :choco-info: **NOTE**
+>
+> Starting with Chocolatey Central Management 0.11.0, the packages installed on a computer can be exported to a `packages.config` file.
+
+From the Computer details page in Central Management, select the `Export to packages.config` button.
```
Comment (gep13) [inline input/en-us/central-management/usage/website/computers.md:52]:
```suggestion
From the Computer details page in Chocolatey Central Management, select the `Export to packages.config` button.
```

**Review by gep13 — CHANGES_REQUESTED** [review]

**Review by st3phhays — APPROVED** [review]
This LGTM!sss

**Comment by gep13** [comment]
@corbob thanks for getting this added!

**Pull request merged**